tests/e2e/e2e_testutils.py

```diff
@@ -143,7 +143,7 @@ def build_test_command(
             "--pod",
             E2ETestRunner.TRESTLEBOT_TEST_POD_NAME,
             "--entrypoint",
-            f"trestlebot-{command_name}",
+            "trestlebot",
```

Comment:
Would this need a `"trestlebot [space][command name]"` for the formatting? I think that may potentially impact the run because instead of just trestlebot it would take `trestlebot create`